spacy/lang/sl/examples.py

```diff
@@ -0,0 +1,18 @@
+"""
+Example sentences to test spaCy and its language models.
+
+>>> from spacy.lang.sl.examples import sentences
+>>> docs = nlp.pipe(sentences)
+"""
+
+
+sentences = [
+    "Apple načrtuje nakup britanskga startupa za 1 bilijon dolarjev",
```

Comment:
I run examples through google translate just to double-check and it suggested that "britanskga" might be a misspelling of "britanskega". Could obviously be nonsense, but thought I'd ask to be sure?

Reply:
Oh yes it is a typo. "britanskega" is the right spelling. Thanks for checking.